crates/core/src/api/body.rs

```diff
@@ -0,0 +1,116 @@
+#[cfg(not(target_arch = "wasm32"))]
```

Comment:
@kearfy Can we use `target_family = "wasm"` instead? Please see [this PR](https://github.com/surrealdb/surrealdb/pull/5357) for details.